package.json

```diff
@@ -28,7 +28,9 @@
     "@types/body-parser": "^1.17.0",
     "@types/express": "^4.16.0",
     "@types/jest": "^23.3.1",
+    "husky": "^6.0.0",
```

Comment:
I don't think husky works this way anymore, either install an older version of husky or follow the new set up instructions.

https://www.npmjs.com/package/husky

Reply:
oh oops you're right i didn't see it pulled in v6

Reply:
updated!